### DISCUSSION TITLE
Counter-Strike 1.6

### DISCUSSION THREAD

**Discussion by asheroto** [seDirector/App#7] [Ideas]
Support for Counter-Strike 1.6

**Comment by asheroto** [comment]
https://store.steampowered.com/stats/Steam-Game-and-Player-Statistics

**Comment by crashzk** [comment]
Good, not if it would be the same system, but there's **Counter-Strike: Condition Zero** too, a lot still played on community servers, especially Zombie, Super Hero and such game modes.

**Comment by asheroto** [comment]
Thanks again for your feedback. This is now an officially planned feature by use of server templates:

https://github.com/seDirector/App/issues/62

**Comment by asheroto** [comment]
I've added this to the [Roadmap](https://github.com/seDirector/App/projects/2#card-89048151) and created an [Issue](https://github.com/seDirector/App/issues/62).
